js/ui/html_editor/utils/toolbar_helper.js

```diff
@@ -109,8 +109,11 @@ function prepareShowFormProperties(module, type) {
         if(!$element?.length) {
             $element = $(getTargetTableNode(module, type));
         }
+        const tableData = module.quill.getModule('table').getTable();
```

Comment:
```suggestion
        const [table, row, cell] = module.quill.getModule('table').getTable() ?? [];
```
And pass further only what you really need